docs/guides/pseudolocalization.rst

```diff
@@ -0,0 +1,57 @@
+==================
+Pseudolocalization
+==================
+
+There is built in support for `pseudolocalization <https://en.wikipedia.org/wiki/Pseudolocalization>`. 
+Pseudolocalization is a method for testing the internationalization aspects 
+of your application by replacing your strings with altered versions 
+and maintaining string readability. It also makes hard coded strings 
+and improperly concatenated strings easy to spot so that they can be properly localized.
+
+  Example:
+  Ţĥĩś ţēxţ ĩś ƥśēũďōĺōćàĺĩźēď
+
+Configuration
+=============
+
+To setup pseudolocalization add :conf:`pseudoLocale` to your lingui :doc:`configuration file </ref/conf>`::
+
+   {
+      "lingui": {
+         "locale": ["en", "pseudo-LOCALE"],
+         "pseudoLocale": "pseudo-LOCALE"
+         "fallbackLocales": {
+            "pseudo-LOCALE": "en"
+         }
+      }
+   }
+
+:conf:`pseudoLocale` option can be any string that is in :conf:`locale` 
+
+Examples: :conf:`en-PL`, :conf:`pseudo-LOCALE`, :conf:`pseudolocalization` or :conf:`en-UK`
+
+Create pseudolocalization
+=========================
+
+PseudoLocale string have to be in :conf:`locales` config as well. 
+Otherwise no folder and no pseudlocalization is going to be created.
+After running ``yarn extract`` verify that the folder has been created.
+The pseudolocalization is automatically created on ``yarn compile`` from messages 
+in order specified in :cli:`Preparing catalogs for production`. 
+In case fallbackLocales has been used, the pseudlocalization is going to be created from translated fallbacklocale.
```

Comment:
improved wording and description in this paragraph

Reply:
```
:cli:`Preparing catalogs for production`
```
This link is not correct. How can I link to cli tutorial?

Reply:
`cli <../ref/cli.html>`_

I found another broken link if you can change it plz, 

### From this:
``@lingui/cli`` provides the ``lingui`` command for extracting, merging and
compiling message catalogs. Follow `setup instructions <../ref/cli>`_ to install required
packages.

### to this 
``@lingui/cli`` provides the ``lingui`` command for extracting, merging and
compiling message catalogs. Follow `setup instructions <../ref/cli.html>`_ to install required
packages.

Reply:
> cli <../ref/cli.html>_

Doesn't link to `Preparing catalogs for production` how can I link to section in cli?

Reply:
Here it is, just like a html anchor 👍🏻 
```
in order specified `in this cli section <../tutorials/cli.html#preparing-catalogs-for-production>`_.

```